src/main/java/uk/ac/ebi/embl/converter/gff3toff/GFF3Mapper.java

```diff
@@ -58,25 +58,19 @@ public Entry mapGFF3ToEntry(GFF3Annotation gff3Annotation) {
         entry = entryFactory.createEntry();
         Sequence sequence = sequenceFactory.createSequence();
 
-        if (!gff3Annotation.getDirectives().getDirectives().isEmpty()) {
-            SourceFeature sourceFeature = this.featureFactory.createSourceFeature();
-            for (GFF3Directives.GFF3Directive directive :
-                    gff3Annotation.getDirectives().getDirectives()) {
-                if (directive.getClass() == GFF3Directives.GFF3SequenceRegion.class) {
-                    GFF3Directives.GFF3SequenceRegion reg = (GFF3Directives.GFF3SequenceRegion) directive;
-                    entry.setPrimaryAccession(reg.accessionId());
+        SourceFeature sourceFeature = this.featureFactory.createSourceFeature();
 
-                    sequence.setVersion(reg.accessionVersion().orElse(1));
-
-                    Location location = this.locationFactory.createLocalRange(reg.start(), reg.end());
-                    Join<Location> compoundJoin = new Join<>();
-                    compoundJoin.addLocation(location);
-                    sourceFeature.setLocations(compoundJoin);
-                }
-            }
-            entry.addFeature(sourceFeature);
+        GFF3SequenceRegion reg = gff3Annotation.getSequenceRegion();
```

Comment:
Rename `reg -> sequenceRegion ` and also set accession to sequence as below:
`sequence.setAccession(reg.accessionId());`